src/metering/metering.rs

```diff
@@ -56,42 +55,59 @@ pub(crate) fn start_metering<T: MeteringClient>(
         }
     });
 
-    (
-        MeteringThreadHandle {
-            _thread_handle: thread,
-        },
-        MeteringRecorder {
+    MeteringRecorder {
+        _thread: thread,
+        sender: MeteringRecorderSender {
             evaluation_event_sender: sender,
         },
-    )
+    }
 }
 
 /// Allows recording of evaluation events.
-/// Communicates with the MeteringThreadHandle, which leads to eventual transmission of recorded evaluations to the server.
+/// Communicates with the thread, which leads to eventual transmission of recorded evaluations to the server.
+#[derive(Debug)]
 pub(crate) struct MeteringRecorder {
+    _thread: ThreadHandle<()>,
+    pub(crate) sender: MeteringRecorderSender,
+}
+
+#[derive(Debug, Clone)]
+pub(crate) struct MeteringRecorderSender {
     evaluation_event_sender: mpsc::Sender<EvaluationEvent>,
 }
 
-impl MeteringRecorder {
-    /// Record the evaluation of a feature or property, for eventual transmission to the server.
-    pub fn record_evaluation(
+impl MeteringRecorderSender {
+    /// Record the evaluation of a property, for eventual transmission to the server.
+    pub fn record_property_evaluation(
```

Comment:
From the consumer perspective, I think it makes total sense to have two different methods instead of making `SubjectId` public. 

I know I suggested the opposite when first reviewing this work, but now I've changed my mind 🤷